src/library.js

```diff
@@ -910,13 +910,15 @@ LibraryManager.library = {
     {{{ makeStructuralReturn(['ret', '0']) }}};
   },
 
-  llvm_ctpop_i32: function(x) {
```

Comment:
For abs, we already redirect to `Math.abs` which should be optimal.

You can make these functions show up in the asm module, look for examples with `__asm: true` in the library JS files. ctpop does look like it could be such a method.